### DISCUSSION TITLE
Good first issues for new developers?

### DISCUSSION THREAD

**Discussion by ehogan** [ESMValGroup/Community#36] [General]
Hi @ESMValGroup/esmvaltool-developmentteam :)

Iris use a [Good First Issue](https://github.com/SciTools/iris/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22) label to identify "good issues to take on if you're just getting started with Iris development". How would everyone feel about doing something like this in ESMValTool / ESMValCore?

In the meantime, does anyone know of any good first issues in either ESMValTool or ESMValCore, please? :)

**Comment by zklaus** [comment]
I think it's a good idea. Perhaps we can go through the list of issues in one @ESMValGroup/technical-lead-development-team meeting?

**Reply by bouweandela** [reply]
Are you looking to get started with some easy issues yourself @ehogan? Most people who contribute to the tool have very specific issues they would like to fix, because they need those. It's quite rare for people to pick up something random.

**Reply by ehogan** [reply]
@bouweandela I wanted to give our new industrial placement something to do that demonstrates working with GitHub and the development workflow process, and I thought it would be great if they could contribute something useful to ESMValTool in the process :)

**Reply by bouweandela** [reply]
Maybe she/he could start with https://github.com/ESMValGroup/ESMValTool/issues/2337, https://github.com/ESMValGroup/ESMValTool/issues/2002? Any contributions that make the documentation easier to understand/navigate for newcomers would also be very welcome.

**Comment by ehogan** [comment]
I just stumbled across [GitHub Docs: Encouraging helpful contributions to your project with labels](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/encouraging-helpful-contributions-to-your-project-with-labels), which recommends applying "the `good first issue` label to issues in your repository to highlight opportunities for people to contribute to your project", so GitHub thinks this is a good idea too! 😉 

When I visit https://github.com/ESMValGroup/ESMValTool/issues, I see: 

<kbd>![image](https://user-images.githubusercontent.com/918142/229476376-2d611491-963b-469d-b4e0-4038ee8b4b98.png)</kbd>

The label itself doesn't appear to be listed at https://github.com/ESMValGroup/ESMValTool/labels, but if we did choose to add it then issues with that label will show up in https://github.com/ESMValGroup/ESMValTool/contribute :)
